frontend/public/components/_edit-yaml.scss

```diff
@@ -1,9 +1,11 @@
 .yaml-editor-header {
   border-bottom: 1px solid #ddd;
   font-size: 16px;
-  height: 45px;
   padding: 10px 20px;
 }
+.yaml-editor-subheader {
```

Comment:
I realize the existing names aren't really following BEM, but this should probably be `.yaml-editor__subheader` (and existing class above `.yaml-editor__header`)

http://getbem.com/naming/

cc @rhamilto